### ISSUE TITLE
can't find linked attachment

### ISSUE THREAD

**Issue by jundurraga** [mickstar/Zoo-For-Zotero#89]
Dear Michel, 
I tried the beta version with linked references and custom path settings. 
I found two issues. First, when setting the location of the custom path, I could not select the folder with all the attachments (the app has granted access to my storage). I can see the folder where I have all the pdfs but I'm not allowed to select it. The only folder I can select without problems is the Download one (using the left-top combo-box option).
To test if I could open a linked pdf in the Download folder, I just copied one of them and tried to open it from Zoo, but I got the following error:
can't find attachment "attachments:name_of_the_file". 
I tried this adding the file using either the standard Zotero option or Zotfile. 
Looking at the database, I can confirm that links are saved in the same way for normal Zotero or Zotfile created links as "attachment:file_name" (in itemAttachmnts table). 
So perhaps this just needs a regexp that gets rid of the "attachment:" string in case of linked items. 
Cheers,
jaime

**Comment by jundurraga** [comment]
I requested a pull that fixed this issue. Please ignore the folder selection issue. It was my table problem.

**Comment by StahlTim** [comment]
Yes, this sounds exactly like the issue I am facing ("attachment:" in the file path)!

Your solution sound / looks promising! Looking forward to the next release!
